docs/button/button.html

```diff
@@ -0,0 +1,218 @@
+                                It appears in front of all screen content, typically as a circular shape with an icon in its center.
+                            </p>
+                            <div class="component-container">
+                                <button class="fab btn-secondary">
+                                    <i class="fas fa-moon"></i>
+                                </button>
+                                <button class="fab btn-danger">
+                                    <i class="fas fa-plus"></i>
+                                </button>
+                                <button class="fab btn-simple">
+                                    <i class="fas fa-check"></i>
+                                </button>
+                                <button class="fab btn-primary">
+                                    <i class="fas fa-cart-arrow-down"></i>
+                                </button>
+                            </div>
+                            <pre>
+                                <code class="code-snippet">
+                                    &#x3C;!-- create FAB by adding perticular classes --&#x3E;
+
+                                    &#x3C;button class=&#x22;fab btn-secondary&#x22;&#x3E;
+                                        &#x3C;i class=&#x22;fas fa-moon&#x22;&#x3E;&#x3C;/i&#x3E;
+                                    &#x3C;/button&#x3E;
+
+                                    &#x3C;button class=&#x22;fab btn-danger&#x22;&#x3E;
+                                        &#x3C;i class=&#x22;fas fa-plus&#x22;&#x3E;&#x3C;/i&#x3E;
+                                    &#x3C;/button&#x3E;
+
+                                    &#x3C;button class=&#x22;fab btn-simple&#x22;&#x3E;
+                                        &#x3C;i class=&#x22;fas fa-check&#x22;&#x3E;&#x3C;/i&#x3E;
+                                    &#x3C;/button&#x3E;
+
+                                    &#x3C;button class=&#x22;fab btn-primary&#x22;&#x3E;
+                                        &#x3C;i class=&#x22;fas fa-cart-arrow-down&#x22;&#x3E;&#x3C;/i&#x3E;
+                                    &#x3C;/button&#x3E;
+                                </code>
+                            </pre>
+                            <div class="docs-footer">
+                                <div class="footer-left">
+                                    <a href="../alert/alert.html" class="btn">
+                                        <i class="fas fa-chevron-left"></i>  Alert
+                                    </a>
+                                </div>
+                                <div class="footer-right">
+                                    <a href="../avatar/avatar.html" class="btn">
+                                      Avatar <i class="fas fa-chevron-right"></i>
+                                    </a>
+                                </div>
+                            </div>
```

Comment:
There is a semantic element `<footer>` use it instead of div with class docs-footer

Reply:
sure will add that tag